src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -286,19 +287,19 @@ def _detect_contact(x: jtp.Array, y: jtp.Array, z: jtp.Array) -> jtp.Array:
         )
 
         # Compute the activation state of the collidable points
-        δ = jax.vmap(_detect_contact)(*position.T)
+        δ = jax.vmap(detect_contact)(*position.T)
```

Comment:
Probably we can use `compute_penetration_data` directly

```suggestion
        δ, δ_dot, _ = contacts.common.compute_penetration_data(p=position, v=velocity, terrain=model.terrain)
```

Reply:
The two functions are not compatible. It's part of the refactoring that will be done in the next PR.